config/project/config.go

```diff
@@ -42,4 +42,10 @@ func Configure(p *config.Provider) {
 			TerraformName: "infisical_project",
 		}
 	})
+
+	p.AddResourceConfigurator("infisical_secret", func(r *config.Resource) {
+		r.Kind = "InfisicalSecret"
```

Comment:
I think for consistency with the other kind values, this should just be `Secret` right? or is that problematic?

Reply:
by consistency I mean like none of the other resources have `infisical` in them